wandb/sdk/wandb_setup.py

```diff
@@ -244,6 +258,10 @@ def _teardown(self, exit_code: int = None):
         self._teardown_manager(exit_code=exit_code)
 
     def _setup_manager(self) -> None:
+        if self._manager:
+            return
+        if self._no_manager:
+            return
```

Comment:
Add a `# fixme:` here and everywhere else to remember to take this out once service is the default.